VERSIONING.md

```diff
@@ -0,0 +1,55 @@
+Rocket Chip Versioning Strategy ** All Strawman to be discussed**
+===============================
+
+Version Naming
+--------------
+
+* Rocket Chip uses a Calendar Versioning release strategy (https://calver.org/).
+* The format is YYYY.MM.DD.MINOR
```

Comment:
```suggestion
* The format is YYYY.MM.DD.MINOR
  * For example, 2020.04.04.0 is the release which occurred on April 4, 2020.
  * 2020.04.04.1 is a release that could occur at any time, but introduces only MINOR changes over the 2020.04.04 release
  * 2020.06.01.0 is a release that occurs on June 1, 2020, with major API breaking changes over the 2020.04.04.X release.
```

Reply:
This does not match the CalVer formatting conventions. Specifically:

> - MM - Short month - 1, 2 ... 11, 12
> - 0M - Zero-padded month - 01, 02 ... 11, 12

You should either update the string to say `YYYY.0M.0D` or update the examples to say `2020.4.4`.